### PR TITLE
Feature/claim

### PR DESCRIPTION
**Pull request by zuccha**

### CODE REVIEW

File: public/css/freelancer.css [hunk 1]
```diff
@@ -190,3 +190,43 @@ ul.tag-list li {
...
+}
+
+#freelancer-claim-form {
+  background-color: ghostwhite;
+  display: flex;
+  flex-flow: column;
+}
+
+#freelancer-claim-form-message {
+    color: red;
```
Comment (larabr) [inline public/css/freelancer.css]: Indentation?

File: test/routes-test/claim.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+var seedDb = require('../../seed_data/seedDb');
+var utils = require('../../seed_data/utils');
+var request = require('supertest');
+
+describe('Claim-post test: ', function() {
+
+  describe('POST /claim', function() {
+
+    before(seed);
+    after(utils.dropDb);
```
Comment (larabr) [inline test/routes-test/claim.js]: change this to also close the connection (see the edit aron did on `develop` on all the other route tests)

File: public/css/freelancer.css (hunk 1 above)
Comment (zuccha) [reply public/css/freelancer.css]: Fixed.

File: test/routes-test/claim.js (hunk 2 above)
Comment (zuccha) [reply test/routes-test/claim.js]: Done.

File: test/nightwatch-tests/freelancer-view.js [hunk 3]
```diff
@@ -40,6 +40,18 @@ module.exports = {
...
       .assert.hidden('div.review-text')
+      // claim
+      .waitForElementPresent('div#freelancer-claim', 1000)
+      .assert.containsText('div#freelancer-claim-status-name', 'NOT VERIFIED')
+      .click('button#freelancer-claim-toggle')
+      .waitForElementPresent('div#freelancer-claim-form', 5000)
+      .click('button#freelancer-claim-btn')
+      .pause(2000)
+      .assert.containsText('div#freelancer-claim-form-message', 'No file was given')
+      .setValue('input#freelancer-claim-form-files', require('path').resolve(__dirname + '/README.md'))
```
Comment (aronfiechter) [inline test/nightwatch-tests/freelancer-view.js]: To get the project root, you can use the field `projectRoot` exported by `test/config.js`

**Review by aronfiechter — COMMENTED** [review]
Looks good.

File: test/nightwatch-tests/freelancer-view.js (hunk 3 above)
Comment (zuccha) [reply test/nightwatch-tests/freelancer-view.js]: Changed.

**Pull request merged**